packages/embark/src/lib/modules/geth/index.js

```diff
@@ -50,23 +60,41 @@ class Geth {
   }
 
   _getNodeState(err, version, cb) {
-    if (err) return cb({name: "Ethereum node not found", status: 'off'});
+    if (err) return cb({ name: "Ethereum node not found", status: 'off' });
 
     let nodeName = "go-ethereum";
     let versionNumber = version.split("-")[0];
     let name = nodeName + " " + versionNumber + " (Ethereum)";
-    return cb({name, status: 'on'});
+    return cb({ name, status: 'on' });
+  }
+
+  _doCheck(cb) {
+    const { rpcHost, rpcPort, wsRPC, wsHost, wsPort } = this.blockchainConfig;
+    if (wsRPC) {
+      return ws(wsHost, wsPort + 10, (err, version) => this._getNodeState(err, version, cb));
+    }
+    rpc(rpcHost, rpcPort + 10, (err, version) => this._getNodeState(err, version, cb));
+  }
+
+  _suppressConnectionOpenError(enable) {
```

Comment:
I just saw that the check does use a JSON RPC call. Can you just confirm that the `connection not open on send` error is indeed printed without that. Since it's not using web3, it shouldn't appear

Reply:
Removed.